L/LASzip/build_tarballs.jl

```diff
@@ -16,24 +17,38 @@ cd $WORKSPACE/srcdir
 
 # Patch to find unordered_map from tr1, logic there doesn't work for MINGW32.
 if [[ "${target}" == *-mingw* ]]; then
-    sed -i '/add_definitions(-DUNORDERED)/d' LASzip/src/CMakeLists.txt;  
+    sed -i '/add_definitions(-DUNORDERED)/d' LASzip/src/CMakeLists.txt;
 fi
 
+# build the DLL
 mkdir LASzip/build
 cd LASzip/build/
-cmake -DCMAKE_INSTALL_PREFIX=$prefix -DCMAKE_CXX_FLAGS="-std=c++11" -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} -DCMAKE_BUILD_TYPE=RelWithDebInfo -DCMAKE_VERBOSE_MAKEFILE=OFF ..
+cmake -DCMAKE_INSTALL_PREFIX=$prefix -DCMAKE_CXX_FLAGS="-std=c++1y" -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} -DCMAKE_BUILD_TYPE=RelWithDebInfo -DCMAKE_VERBOSE_MAKEFILE=OFF ..
```

Comment:
Why are you changing the C++ standard here?
```suggestion
cmake -DCMAKE_INSTALL_PREFIX=$prefix -DCMAKE_CXX_FLAGS="-std=c++14" -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} -DCMAKE_BUILD_TYPE=RelWithDebInfo -DCMAKE_VERBOSE_MAKEFILE=OFF ..
```
Don't yse the `c++1y` specifier, use `c++14` (the `1y` specifier is deprecated and also not obvious what the actual standard is). Also, if you want C++14, you probably also should introduce a higher GCC minimum version to ensure you get the language features properly. So, please add
```
preferred_gcc_version = v"5"
```
to the `build_tarballs` function call at the bottom of this file.

Reply:
When I was testing locally LAStools wouldn't build in c++11 as it had some namespace definitions from c++14. I'll try updating the gcc version though! Thanks :)